### PR TITLE
AB#95731 AB#79507  Create view for organisation list

### PR DESCRIPTION
**Pull request by dawid-pawulski**
### 🛠 Changes being made

Create view for organisation list

### 📸 Screenshots (optional)

image

### 🏎 Quality check

- [x] Have you performed local tests?
- [x] Are integration tests passing locally?
- [x] Have you checked WCAG (included screenshot)

### CODE REVIEW

File: src/HE.Investments.Account.IntegrationTests/Pages/MainPagesUrl.cs [hunk 1]
```diff
@@ -10,6 +10,8 @@ public static class MainPagesUrl
 
     public const string Dashboard = "/user-organisation";
 
+    public const string OrganisationList = "/user-organisation/list";
```
Comment (Carq) [inline src/HE.Investments.Account.IntegrationTests/Pages/MainPagesUrl.cs:13]: -> `user-organisations`

File: src/HE.Investments.Account.WWW/Controllers/UserOrganisationController.cs [hunk 2]
```diff
@@ -119,9 +119,32 @@ public async Task<IActionResult> ChangeOrganisationDetails(OrganisationDetails m
     }
 
     [HttpGet("list")]
-    public IActionResult UserOrganisationsList()
+    public async Task<IActionResult> UserOrganisationList()
```
Comment (Carq) [inline src/HE.Investments.Account.WWW/Controllers/UserOrganisationController.cs:122]: Create seperate controller for handling organisations.

**Review by Carq — APPROVED** [review]

**Review by jjankowski87 — APPROVED** [review]

File: src/HE.Investments.Account.IntegrationTests/Pages/MainPagesUrl.cs (hunk 1 above)
Comment (dawid-pawulski) [reply src/HE.Investments.Account.IntegrationTests/Pages/MainPagesUrl.cs:13]: i will change it in the next pr

File: src/HE.Investments.Account.WWW/Controllers/UserOrganisationController.cs (hunk 2 above)
Comment (dawid-pawulski) [reply src/HE.Investments.Account.WWW/Controllers/UserOrganisationController.cs:122]: i will change it in the next pr

**Pull request merged**